### DISCUSSION TITLE
Question: What is the optimal way to check the same cronjob on a few servers?

### DISCUSSION THREAD

**Discussion by vitalykarasik** [healthchecks/healthchecks#481] [Q&A]
I have certain cronjobs which are running on a few different servers. For example - "Clean job", which runs on all application server.  I'm curious is there an elegant way for their monitoring?
If I use the same job ID, I suspect that Healthcheck  may miss failed job.
If I use different job IDs, I need to add them to Healthcheck and my Chef recipies - not so elegant.
Please suggest.

Thanks for your great service,
Vitaly

**Comment by cuu508** [comment]
Having multiple checks, one per server, is the way to go.

You can look at API and API wrappers for automating deployment. For example, here's a script that creates the check if it does not exist:

https://dev.to/15five/managing-health-checks-at-scale-4731

**Reply by vitalykarasik** [reply]
Thank you, it's interesting direction!
BTW, I posted about my experience with Healthcheck https://www.linkedin.com/posts/vkarasik_free-cron-job-monitoring-healthchecksio-activity-6764471140724293632-0otU, bu didn't know how to tag you or your project. I'm enjoying  to use Healthcheck and recommend it to all my colleagues :-)

**Reply by cuu508** [reply]
Happy to hear you are enjoying the service, and thanks for the shout-out!

A couple months back I deleted my LinkedIn account – not being able to tag me is a good confirmation the delete actually worked ;-)
